eahub/localgroups/views.py

```diff
@@ -1,14 +1,37 @@
 import json
-from datetime import datetime 
+from datetime import datetime
 
 from django.urls import reverse_lazy
 from django.shortcuts import render, redirect, get_object_or_404
 from django.contrib.auth.decorators import login_required
-
+from django.core.mail import send_mail
+from django.contrib.sites.shortcuts import get_current_site
+from .forms import ClaimGroupForm
 from .models import LocalGroup as Group
 
 def view_group(request, slug):
     group = get_object_or_404(Group, slug=slug)
+
     return render(request, 'eahub/group.html', {
         'group': group
     })
+
+def claim_group(request, slug):
+    group = get_object_or_404(Group, slug=slug)
+    if request.method == 'POST':
+        if request.user.is_anonymous:
+            return render(request, 'eahub/claim_group_error.html')
+        else:
+            subject = "EA Group claimed: {0}".format(group.name)
+            user_eahub_url = "https://{0}/profile/{1}".format(get_current_site(request).domain,request.user.profile.slug)
+            message = """ User <a href="{0}">{1}</a> wants to claim group {2}.
+                          Please check whether the user is eligible to claim this group
```

Comment:
Those leading whitespaces will appear in the email. One possible option is to move this text into a template and get it with [`render_to_string`](https://docs.djangoproject.com/en/2.1/topics/templates/#django.template.loader.render_to_string).

Reply:
added template